test/functional/wiki_controller_test.rb

```diff
@@ -193,7 +193,7 @@ def teardown
         id: 'chicago'
         }
 
-    assert_equal flash[:notice], "Please post a question or other content before editing the wiki. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
+    assert_equal flash[:notice], "You can create the wiki once your research note/question is approved by moderators. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
```

Comment:
So strange that this failed!? I don't understand -- we changed the test here too!

```
Progress: |=== FAIL WikiControllerTest#test_disallow_viewing_edit_wiki_page_for_first-timers (38.87s)
        --- expected
        +++ actual
        @@ -1 +1 @@
        -"Please post a question or other content before editing the wiki. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
        +"You can create the wiki once your research note/question is approved by moderators. Click <a href='https://publiclab.org/notes/tester/04-23-2016/new-moderation-system-for-first-time-posters'>here</a> to learn why."
        test/functional/wiki_controller_test.rb:196:in `block in <class:WikiControllerTest>'
```

Reply:
Aha! Ok, we actually have to make the change twice in the controller. I'll make a suggestion!!!